src/test/java/core/basesyntax/service/activities/BalanceHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.activities;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.fruitrecord.FruitRecord;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static final String[] normalRecord = new String[]{"b", "cherry", "100"};
+    private static final String[] normalRecord1 = new String[]{"b", "cherry", "50"};
+    private static final String[] incorrectRecord1 = new String[]{"HelloWorld!", "cherry", "50"};
+    private static final ActivityHandler balanceHandler = new BalanceHandler();
+    private static String expected;
+
+    @Before
+    public void setUp() throws Exception {
+        Storage.fruitsQuantity.clear();
+    }
+
+    @Test
+    public void balanceHandler_addNewBalance_Ok() {
+        FruitRecord fruitRecord = new FruitRecord(normalRecord);
+        balanceHandler.apply(fruitRecord);
+        assertTrue("Can't write this record to db " + Arrays.toString(normalRecord),
+                Storage.fruitsQuantity.containsKey(fruitRecord.getFruit())
+                && Storage.fruitsQuantity.containsValue(fruitRecord.getAmount()));
```

Comment:
I think it's better to get actual Entry of `Map<String, Integer> fruitsQuantity` and compare with expected. What if your service save more than 1 fruit somehow, than `contains` this is not a very good test

Reply:
I think, that will solve the problem
&& Storage.FRUITS_QUANTITY.size() == 1)
and I agree, to use "contains" it is not very good solution